index.html

```diff
@@ -594,52 +596,57 @@
 
-[#Burley2018] derived (via fitting Monte Carlo simulations) an approximate relationship between the specified surface diffusion profile width $\mathbf{r}$ and multi-scatter albedo $\mathbf{A}$, and the physical extinction coefficient $\boldsymbol{\mu}_t$ which generates that profile width: $\boldsymbol{\mu}_t = 1/(S(\mathbf{A})\mathbf{r})$, where the scale-factor $S(\mathbf{A})$ is given by:
+To define the meaning of the observed **`subsurface_color`** $\mathbf{C}$ in terms of the underlying medium, the reflection from the medium needs to be separated from the Fresnel reflection at the dielectric interface. To make this precise, we denote by $\mathbf{E}_\mathrm{subsurface}$ the directional reflectance along the the normal direction (according to the definition of equation [directional-reflectance-definition]).
+This can be broken into two components:
 \begin{equation}
-S(\mathbf{A}) = 4.012 - 15.21 \mathbf{A} + 32.34 \mathbf{A}^2 - 34.68 \mathbf{A}^3 + 13.91 \mathbf{A}^4 \ .
+\mathbf{E}_\mathrm{subsurface} = \mathbf{E}_\mathrm{spec} + \mathbf{E}_\mathrm{multi-scatter} \ .
 \end{equation}
-Similarly the RGB single-scattering albedo $\boldsymbol{\alpha}$ is approximately given by
-\begin{equation}
-\boldsymbol{\alpha}(\mathbf{A}) = 1 - e^{-11.43 \mathbf{A} + 15.38 \mathbf{A}^2 - 13.91 \mathbf{A}^3} \ .
+Here $\mathbf{E}_\mathrm{spec}$ is the normal-direction reflectance of all energy reflected from the dielectric interface _without_ transmission. While $\mathbf{E}_\mathrm{multi-scatter}$ (the _multiple-scattering albedo_) is the normal-direction reflectance of all remaining reflected energy due to transmission through the interface, multiple scattering in the subsurface medium, and transmission back out.
+We then _define_ the **`subsurface_color`** $\mathbf{C}$ to parametrize the fraction of the energy transmitted through the interface that is transmitted back out and observed, i.e.:
+\begin{equation} \label{subsurface_albedo_constraint}
+\mathbf{E}_\mathrm{multi-scatter} = \left( 1 - \mathbf{E}_\mathrm{spec} \right) \mathbf{C} \ .
 \end{equation}
-The physical RGB scattering coefficient is then given by $\boldsymbol{\mu}_s = \boldsymbol{\alpha}\,\boldsymbol{\mu}_t$.
+Thus since $0 \le \mathbf{C} \le 1$, $\mathbf{E}_\mathrm{subsurface} \le 1$ so energy is always conserved, and if $\mathbf{C}=1$ then $\mathbf{E}_\mathrm{subsurface}=1$ which guarantees that a white **`subsurface_color`** passes the furnace test. According to this definition, the **`subsurface_color`** $\mathbf{C}$ is the observed reflection color (viewed at normal incidence under uniform illumination) in areas where the Fresnel reflection is negligible, and otherwise the observed color is a blend of $\mathbf{C}$ with the gray Fresnel reflection that conserves total reflected energy.
```

Comment:
> (viewed at normal incidence under uniform illumination)

We should also mention the semi-infinite homogeneous slab assumption. (Without it, the albedo remapping would have to take the geometry into account, and then this varying remapping would cancel out some of the interesting effects that subsurface scattering produces, such as thin regions looking less saturated than thick regions.)

Reply:
Yep good point, that does need to be made explicit. Arguably this is implicit in the definition of the Slab concept, which we introduced saying:

>Each slab consists of a homogeneous dielectric or conducting medium $V$ with conceptually unbounded extent horizontally and a finite vertical extent ...

But it doesn't hurt to re-emphasize it. This makes the wording sound a bit pedantic and technical, but it's worth stating to be fully correct. 

In https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171/commits/59ed616715b90877bdeddbf1f7935fc78956bf56 I made the language more precise as follows:

>We then _define_ the **`subsurface_color`** $\mathbf{C}$ to parametrize, assuming correct physical light transport and infinite homogeneous slab geometry, the fraction of the energy transmitted through the interface that is transmitted back out and observed ...

Probably we should add similar clarification in the glossy-diffuse section as well.

Reply:
Good point that the whole model has a slab assumption. It makes sense to clarify it in this subsurface section anyway since the subsurface light transport controlled by the subsurface parameters happens at the macroscopic scale.